spec/lib/roadie/document_spec.rb

```diff
@@ -98,6 +104,15 @@ module Roadie
         }
         expect { document.transform }.to_not raise_error
       end
+
+      context 'when save as xhtml flag is present' do
+        it "does not escape some HTML symbols" do
+          document = Document.new "<body><a href='https://google.com/{{hello}}'</body>"
```

Comment:
Sure, but I would not expect `{{hello}} ` to be in this format: `foo'>` to correctly close the `<a>` element.